worksheet_A/TerminalHacking.cpp

```diff
@@ -7,6 +7,18 @@
 const int wordLength = 5;
 const int numberOfWords = 15;
 
+int getLikeness(std::string guessword, std::string password)
+{
+	int score = 0;
+
+	for (int i = 0; i < wordLength; i++) {
```

Comment:
Follow a consistent coding style -- i.e. if the rest of the code puts the `{` on the next line, yours should too